content/news/XXXX-XX-XX-mixxx-2-3-5-released.md

```diff
@@ -0,0 +1,62 @@
+title: Mixxx 2.3.5 released
+authors: Nikolaus Einhauser
+tags: 2.3, 2.3.5, release announcement
+comments: yes
+status: draft
+
+Another bugfix release, this soon?
+
+Yes, unfortunately it turned out [2.3.4 introduced a couple serious
+bugs](https://github.com/mixxxdj/mixxx/issues?q=is%3Aissue+is%3Aclosed+milestone%3A2.3.5+label%3Aregression),
+that required prompt action to fix them and to rectify that sub-par release.
+
+The most serious issue we fixed was a regression that caused an empty waveform
+overview after loading a track. This issue has been fixed in version 2.3.5,
+along with several other bug fixes and improvements.
+
+Other notable changes include requesting microphone and line-in access
+permission on macOS, allowing explicit selection of buffers of 2048 and 4096
+frames/period in the JACK API, making the Beat FX section of the Pioneer DDJ-400
+more intuitive, and fixing a visual issue with the the VU peak indicators in the
+Tango skin.
+
+Overall, version 2.3.5 aims to improve stability and performance while
+addressing the issues introduced in the previous release. Users are encouraged
+to update to this version as soon as possible to ensure the best possible
+experience while using Mixxx.
+
+Here is the full Changelog if you're interested:
+
+* Fix empty waveform overview after loading a track (Mixxx 2.3.4 regression)
+  Fixed by [#11333](https://github.com/mixxxdj/mixxx/pull/11333)
+  [#11359](https://github.com/mixxxdj/mixxx/pull/11359)
+  [#11344](https://github.com/mixxxdj/mixxx/issues/11344)
+* Fullscreen: Fix a crash that occurs on Linux after enabling fullsceen and using menu
+  shortcuts e.g. Alt-F.
+  [#11328](https://github.com/mixxxdj/mixxx/pull/11328)
+  [#11320](https://github.com/mixxxdj/mixxx/issues/11320)
+* Fullscreen: Rebuild & reconnect menu only on desktops with global menu
+  [#11350](https://github.com/mixxxdj/mixxx/pull/11350)
+* macOS: Request Microphone and line-in access permission.
+  [#11367](https://github.com/mixxxdj/mixxx/pull/11367)
+  [#11365](https://github.com/mixxxdj/mixxx/issues/11365)
+* JACK API: Allow to explicit select buffers of 2048 and 4096 frames/period. They are not
```

Comment:
My gut feeling tells me that "allow to" can't be used that way. Also it should be "explicitly" since it's used as an adverb. Perhaps

```suggestion
* JACK API: Allow explicitly selecting buffers of 2048 and 4096 frames/period. They are not
```

or maybe

```suggestion
* JACK API: Allow for explicit selection of buffers with 2048 and 4096 frames/period. They are not
```

Reply:
These have been copy-pasted from the changelog, I'd prefer if the different copies of the changelog don't diverge.

Reply:
Sure, being consistent makes sense, so feel free to close this, this was just a minor thing I've noticed